packages/eslint-plugin/configs/recommended.js

```diff
@@ -13,12 +13,33 @@ const { config: localPrettierConfig } =
 const prettierConfig = { ...defaultPrettierConfig, ...localPrettierConfig };
 
 module.exports = {
+	settings: {
+		'import/resolver': {
```

Comment:
https://github.com/benmosher/eslint-plugin-import#importcore-modules

Cold we set `react` as core module for TS files to avoid having to set it as a package dependency or is it necessary for TS?

Reply:
It's necessary for TS https://stackoverflow.com/questions/55198502/using-eslint-with-typescript-unable-to-resolve-path-to-module

Reply:
The resolver for file extension is necessary. That's one thing. 

I'm wondering if you need to explicitly declare an import for a package with only types as a dependency of the package that does it. In this example, you import only types from the `react` package. Does `react` need to be declared as a dependency of the package for TypeScript to work correctly when the package is consumed from npm.

Reply:
Ohhh sorry, I misunderstood your original question.

I think yes, we can set `react` in that option and remove it from peerDependencies.